Formula/bach.rb

```diff
@@ -0,0 +1,108 @@
+require "language/go"
+
+class Bach < Formula
+  desc "Command-line for the Compose API"
+  homepage "https://github.com/compose/bach"
+  url "https://github.com/compose/bach.git", :tag => "0.0.10"
```

Comment:
If there aren't submodules or anything requiring a clone, we prefer the tarball: https://github.com/compose/bach/archive/0.0.10.tar.gz

If we do need to fetch it with git, add a `revision` as well so we can catch if it's re-tagged.